0.17/files/docker-entrypoint.sh

```diff
@@ -44,23 +44,23 @@ if [ "$(id -u)" = '0' ]; then
   SU_EXEC="su-exec factorio"
 fi
 
-if ! find -L $SAVES -iname \*.zip -mindepth 1 -print | grep -q .; then
+if ! find -L "$SAVES" -iname \*.zip -mindepth 1 -print | grep -q .; then
   # Generate a new map if no save ZIPs exist
   ${SU_EXEC} /opt/factorio/bin/x64/factorio \
-    --create $SAVES/_autosave1.zip  \
-    --map-gen-settings $CONFIG/map-gen-settings.json \
-    --map-settings $CONFIG/map-settings.json
+    --create "$SAVES/_autosave1.zip" \
+    --map-gen-settings "$CONFIG/map-gen-settings.json" \
+    --map-settings "$CONFIG/map-settings.json"
 fi
 
-exec ${SU_EXEC} /opt/factorio/bin/x64/factorio \
-  --port $PORT \
+exec "${SU_EXEC}" /opt/factorio/bin/x64/factorio \
```

Comment:
```suggestion
exec ${SU_EXEC} /opt/factorio/bin/x64/factorio \
```

Reply:
Can't reproduce that but I changed some other things around. Can you try it with the new commit again, please?